ibm/service/database/resource_ibm_database.go

```diff
@@ -424,6 +435,71 @@ func ResourceIBMDatabaseInstance() *schema.Resource {
 					},
 				},
 			},
+			"group": {
+				Type:          schema.TypeSet,
+				Optional:      true,
+				ConflictsWith: []string{"members_memory_allocation_mb", "members_disk_allocation_mb", "members_cpu_allocation_count", "node_memory_allocation_mb", "node_disk_allocation_mb", "node_cpu_allocation_count", "node_count"},
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"group_id": {
```

Comment:
Do we have any valid values accepted here like "members" if yes can we add validation?